janus_core/helpers/utils.py

```diff
@@ -229,3 +231,87 @@ def dict_remove_hyphens(dictionary: dict) -> dict:
         if isinstance(value, dict):
             dictionary[key] = dict_remove_hyphens(value)
     return {k.replace("-", "_"): v for k, v in dictionary.items()}
+
+
+def results_to_info(
+    struct: Atoms,
+    *,
+    properties: Collection[Properties] = (),
+    invalidate_calc: bool = True,
+) -> None:
+    """
+    Copy or move MLIP calculated results to info.
```

Comment:
What does it mean to be in "info"?

Reply:
The `Atoms.info` dict. I can make it clearer in the various places I reference it